src/config.c

```diff
@@ -160,6 +290,13 @@ int handler(void *user, const char *section, const char *name,
 		}
 	}
 
+	if (CHECK_SECTION("general")) {
+		if (MATCH("general", "log_filepath")) {
+			ini_config->general_config->log_filepath =
+				strdup(value);
+		}
+	}
```

Comment:
Move to the top of the function, the `[general]` section is more likely to be on top of the configuration file.